<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QDialog" name="Options">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="1" column="0">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>106</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_apply">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>&amp;Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_connection">
      <attribute name="title">
       <string>Connection</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="title">
          <string>Connection Target</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0" colspan="2">
           <widget class="QRadioButton" name="radioButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Digital Refractor (CDR)</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="autoRepeat">
             <bool>false</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>16</width>
              <height>61</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Protocol Version</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>System ID</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="comboBox_system_id">
              <item>
               <property name="text">
                <string>0</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>4</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>6</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>7</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBox_protocol_ver">
              <item>
               <property name="text">
                <string>1.0</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1.1</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>80</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="title">
          <string>Port Connection</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Port</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Baud Rate</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBox_speed">
            <item>
             <property name="text">
              <string>9600</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>14400</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>19200</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>38400</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>57600</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>115200</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox_port">
            <property name="editable">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>/dev/ttyUSB0</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>/dev/ttyUSB1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>/dev/ttyUSB2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>/dev/ttyUSB3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>/dev/ttyS0</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>/dev/ttyS1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>/dev/ttyS2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>/dev/ttyS3</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="title">
          <string>Startup &amp;&amp; Connect</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBox_auto_connect">
            <property name="text">
             <string>Connect to port when CDRMate starts up</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_receive_transfer">
      <attribute name="title">
       <string>Data Receiving &amp;&amp; Transfer</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Data Receiving Option</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Automatic Export</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QRadioButton" name="radio_export_on">
            <property name="text">
             <string>ON</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QRadioButton" name="radio_export_off">
            <property name="text">
             <string>OFF</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="4" colspan="2">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>92</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Export Directory</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="4">
           <widget class="QLineEdit" name="lineEdit_export">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QPushButton" name="pushButton_export">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Data Transfer Option</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Import Directory</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="4">
           <widget class="QLineEdit" name="lineEdit_import">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QPushButton" name="pushButton_import">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Automatic Transfer</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QRadioButton" name="radio_transfer_on">
            <property name="text">
             <string>ON</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QRadioButton" name="radio_transfer_off">
            <property name="text">
             <string>OFF</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="4" colspan="2">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>92</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>193</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Options</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>391</x>
     <y>488</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Options</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>391</x>
     <y>488</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_protocol_ver</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Options</receiver>
   <slot>data_modified()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_system_id</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Options</receiver>
   <slot>data_modified()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_port</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Options</receiver>
   <slot>data_modified()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>-73</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_export_on</sender>
   <signal>toggled(bool)</signal>
   <receiver>Options</receiver>
   <slot>data_modified()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_export_off</sender>
   <signal>toggled(bool)</signal>
   <receiver>Options</receiver>
   <slot>data_modified()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_export</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Options</receiver>
   <slot>data_modified()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_import</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Options</receiver>
   <slot>data_modified()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>-58</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_transfer_on</sender>
   <signal>toggled(bool)</signal>
   <receiver>Options</receiver>
   <slot>data_modified()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>-34</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_transfer_off</sender>
   <signal>toggled(bool)</signal>
   <receiver>Options</receiver>
   <slot>data_modified()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_apply</sender>
   <signal>clicked()</signal>
   <receiver>Options</receiver>
   <slot>conf_apply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>477</y>
    </hint>
    <hint type="destinationlabel">
     <x>-20</x>
     <y>475</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_speed</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Options</receiver>
   <slot>data_modified()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_import</sender>
   <signal>clicked()</signal>
   <receiver>Options</receiver>
   <slot>choose_dir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_export</sender>
   <signal>clicked()</signal>
   <receiver>Options</receiver>
   <slot>choose_dir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_auto_connect</sender>
   <signal>toggled(bool)</signal>
   <receiver>Options</receiver>
   <slot>data_modified()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>387</y>
    </hint>
    <hint type="destinationlabel">
     <x>469</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>data_modified()</slot>
  <slot>conf_apply()</slot>
  <slot>choose_dir()</slot>
 </slots>
</ui>
